### DISCUSSION TITLE
ROI selections, black bars

### DISCUSSION THREAD

**Discussion by wonder99** [springbok/MLM2PRO-GSPro-Connector#39] [General]
I question the need to remove black bars on the mirrored screen.  I've never found that necessary and can't see a reason why it would matter.
I also think the screen mirror app should be maximized behind GSPRO always.  And for sure, when picking the ROI locations, that screen might as well be maximized too, for better visibility/accuracy.

**Comment by springbok** [comment]
Yea I was going to test that I don't think it makes a difference either but completely forgot about checking it.
